frontend/cypress/support/helper/dom-helper/pages/overviewPage.ts

```diff
@@ -187,6 +188,27 @@ export default class CyOverviewPage extends Page {
     return new ObjectiveDialog();
   }
 
+  completeObjective(title: string) {
+    this.addObjective()
+      .fillObjectiveTitle(title)
+      .submit();
```

Comment:
Even tho is is technically all right pls update this so the implementation alligns with the way we call the other menu options

Reply:
OR 
check if it would probably be more readable to create for every menu entery a helper method like you did here